### DISCUSSION TITLE
Similar editor for linux

### DISCUSSION THREAD

**Discussion by SolMi-Sera** [Acode-Foundation/Acode#525] [Q&A]
I'm loving every about Acode, from it's working as expected right out of the box with no setup, no need for tutorials,  intuitive interface, and all of this while leaving nothing to be desired. The only thing I wish this had, is a Linux counterpart. 

Can anyone recommend the closest editor they've found to Acode, for Linux? 

Vs code was just too big for me, and being self taught, I don't have the time to on-board another learning curve so, if something like Acode exists out there for Linux, it would be a truly welcome addition for me.

**Comment by deadlyjack** [comment]
Ok, I'll try to build it for linux.

**Reply by SolMi-Sera** [reply]
Wow, that'd be huge! Im happy to contribute in any way I can if it helps this become a reality 🤓

**Comment by malijaffri** [comment]
You could try [Sublime Text](https://www.sublimetext.com/). I actually found Acode while looking for a Sublime-Text–like experience for Android.

**Reply by SolMi-Sera** [reply]
Looks promising, I'll give it a go today. Thank you for the recommendation.

**Comment by ByteJoseph** [comment]
I want to know your current opinion
